silkworm/node/stagedsync/remote_client.cpp

```diff
@@ -29,7 +29,7 @@ namespace silkworm::execution {
 using namespace std::chrono;
 using namespace boost::asio;
 
-static void serialize_header(const BlockHeader& bh, ::execution::Header* header) {
+/*static void serialize_header(const BlockHeader& bh, ::execution::Header* header) {
```

Comment:
I assume that this needs extra work outside of this PR scope, therefore it is commented and not deleted?

Maybe add a TODO comment like in some other cases?